README.md

```diff
@@ -16,15 +16,15 @@ Mendix 7.2
 
 [https://badgebutton.mxapps.io/](https://badgebutton.mxapps.io/)
 
-![1](https://raw.githubusercontent.com/mendixlabs/badge-button/v1.1.0/assets/demo.png)
+![1](https://raw.githubusercontent.com/mendixlabs/badge-button/v1.1.1/assets/demo.png)
```

Comment:
The `![1]` is the alt text, should actually put a sensible value into it. Describing the content of the image.